nasa/earthdata-search
# EDSC-3422: Implements collection subscriptions

# Overview

### What is the feature?

Implements collections subscriptions and improves the display of the query parameters on the subscriptions page

### What is the Solution?

Existing subscriptions functionality was changed to support collection subscriptions

### What areas of the application does this impact?

Collection and granules subscription panels

# Testing

### Reproduction steps

- **Environment for testing:** SIT
- **Collection to test with:** Any

1. Change search parameters
2. Click "subscriptions"
3. Verify any existing collection subscriptions display in the list
4. Verify that the current query parameters are displayed
5. Create a subscription
6. Verify the new subscription appears in the list
7. Verity that the subscription count on the subscription button is accurate

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
